### PR TITLE
[ember] add dependencies to root application

### PR DESCRIPTION
**Pull request by gabrielcsapo**
Issue: #4289

## What I did

Add dependencies to the top level @storybook/ember application

### CODE REVIEW

File: app/ember/package.json [hunk 1]
```diff
@@ -28,12 +28,15 @@
     "@babel/runtime": "^7.0.0",
     "@ember/test-helpers": "^0.7.25",
     "@storybook/core": "4.0.0-alpha.24",
+    "common-tags": "^1.8.0",
+    "moment": "^2.22.2",
```
Comment (igor-dv) [inline app/ember/package.json]: I think `moment` is not needed here. It comes from knobs AFAIR.

File: app/ember/package.json [hunk 2]
```diff
@@ -28,12 +28,15 @@
     "@babel/runtime": "^7.0.0",
     "@ember/test-helpers": "^0.7.25",
     "@storybook/core": "4.0.0-alpha.24",
+    "common-tags": "^1.8.0",
```
Comment (igor-dv) [inline app/ember/package.json]: Why is this moved?

Comment (gabrielcsapo) [reply app/ember/package.json]: It is used in @storybook/ember and that's it.

File: app/ember/package.json (hunk 1 above)
Comment (gabrielcsapo) [reply app/ember/package.json]: will remove, it was being required when I tried to install

File: app/ember/package.json (hunk 2 above)
Comment (igor-dv) [reply app/ember/package.json]: Doesn't ember itself depend on this dep?

Comment (gabrielcsapo) [reply app/ember/package.json]: no, it is used for outputting error information in the render function

**Review by igor-dv — APPROVED** [review]

**Review by pksunkara — APPROVED** [review]

**Comment by fran-worley** [comment]
@igor-dv since adding react and react-dom to my app dependencies it is now being recognised as a react app by `storybook init`. 

Does the order of if statements in the app detection need reworking so ember is detected before react?

**Comment by igor-dv** [comment]
@Keraito is working on a CLI improvement where you will be able to pass a flag of the wanted framework. Probably `react` is one of the first apps to be checked...

**Comment by fran-worley** [comment]
@igor-dv I figured a simple solution would be to move the react clause to the end [here](https://github.com/storybooks/storybook/blob/master/lib/cli/lib/detect.js#L57-L62)

As I side note I see that ember isn't included [here](https://github.com/storybooks/storybook/blob/master/lib/cli/lib/detect.js#L117) is that likely to cause issues for full ember support?

**Comment by igor-dv** [comment]
>  I figured a simple solution would be to move the react clause to the end here

yes, it will probably help, want to PR ?

> As I side note I see that ember isn't included here is that likely to cause issues for full ember support?

I am less familiar with the CLI part. @Keraito @gabrielcsapo do you have something to say about this ?

**Comment by fran-worley** [comment]
@igor-dv compiling PR now.

> As I side note I see that ember isn't included here is that likely to cause issues for full ember support?

Having now added 'ember' to that list, it is correctly detecting when storybook is already setup so I think it needs to be in there and I'll add to my PR in a separate commit.
